_rules/element-lang-matches-default-off6ek.md

```diff
@@ -0,0 +1,337 @@
+
+- This rule assumes that [grandfathered tags][] are not used as these will not be recognized as [valid language tags][].
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+This rule ignores elements that are nested inside `iframe`. `iframe` are intended to provide a layer of isolation, and it is not clear how language is inherited in nested browsing contexts. Incorrect `lang` attribute inside `iframe` are still likely to cause accessibility issues even though this rule won't flag them.
+
+- Related rule:
+  - [_Element with `lang` Attribute Has Valid Language Tag_](https://act-rules.github.io/rules/de46e4)
+- [Understanding Success Criterion 3.1.2: Language of Page][usc312]
+- [H58: Using language attributes to identify changes in the human language](https://www.w3.org/WAI/WCAG21/Techniques/html/H58)
+- [BCP 47: Tags for Identifying Languages](https://www.ietf.org/rfc/bcp/bcp47.txt)
+- [The `lang` and `xml:lang` attributes](https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes)
+
+In all examples, the `html` element has itself a `lang` attribute in order to make sure that the examples do not fail [Success Criterion 3.1.1 Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page). These `html` elements are, however, never applicable because they are not descendants of a `body` element, and the example descriptions do not mention them further.
+
+Similarly, all examples contain a short text in the page language without any `lang` attribute in order to make the `lang` attribute on other elements useful. Especially, Technique [H58: Using language attributes to identify changes in the human language](https://www.w3.org/WAI/WCAG21/Techniques/html/H58) is about changes of language.
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `span` element has a `lang` [attribute value][] of `nl` (Dutch), which matches its [default language][]. The default language is Dutch because all words are in Dutch.
+
+```html
+<html lang="en">
+	<head>
+		<title>Dutch idioms</title>
+	</head>
+	<body>
+		<p>
+			The Dutch phrase <span lang="nl">"Hij ging met de kippen op stok"</span> literally translates into "He went to
+			roost with the chickens", but it means that he went to bed early.
+		</p>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+The second `p` element has a `lang` [attribute value][] of `en` (English), which matches its [default language][]. The default language is English because all but a few words are in English.
+
+```html
+<html lang="nl">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="en">
+			The Dutch phrase "Hij ging met de kippen op stok" literally translates into "He went to roost with the chickens",
```

Comment:
I would fail 3.1.2 here. There is a passage where the correct language cannot be programmatically determined.